.gitignore

```diff
@@ -51,3 +51,6 @@ docs/
 # demo
 demo/dist/
 demo/node_modules
+
+# package-lock
```

Comment:
it's ok but shouldn't we be building it with yarn thus no need for package-lock.json?

Reply:
Yep - that's correct. I built it with `yarn` but IntelliJ IDEA generated a `package-lock.json`, so I added the exclusion just in case

Reply:
it is fine then